<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="submission_table">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>661</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Submission ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Project ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Project Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Submission Date</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Submission Panel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletesubmissionbtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>370</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Submission</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addsubmissionbtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Submission</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subidedit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>330</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="placeholderText">
     <string>Submission ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pidedit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="placeholderText">
     <string>Project ID</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="duedateedit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>410</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_accepted">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>330</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Accepted</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_rejected">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Rejected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changestatusbtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Change Status</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>310</y>
      <width>16</width>
      <height>171</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="goback">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>281</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
